src/bodypart.h

```diff
@@ -274,6 +286,8 @@ class bodypart
 
         void set_encumbrance_data( const encumbrance_data &set );
 
+        void set_mut_drench( std::pair<water_tolerance, int> set );
```

Comment:
Perhaps splitting the pair argument into the two named arguments would be more readable?